src/nbl/video/COpenGL_LogicalDevice.h

```diff
@@ -314,6 +318,12 @@ class COpenGL_LogicalDevice : public IOpenGL_LogicalDevice
         m_threadHandler.waitForRequestCompletion<SRequestRegenerateMipLevels>(req);
     }
 
+    void waitIdle() override
+    {
+        SRequestWaitIdle params;
+        auto& req = m_threadHandler.request(std::move(params));
+        m_threadHandler.waitForRequestCompletion<SRequestWaitIdle>(req);
```

Comment:
I think `glFinish` affects only the current context... you'd have to post a request for a `glFinish` for every single queue and swapchain as well.

Reply:
done?